src/main/java/com/google/api/tools/framework/aspects/versioning/model/VersionAttribute.java

```diff
@@ -16,14 +16,14 @@
 
 package com.google.api.tools.framework.aspects.versioning.model;
 
+import com.google.api.tools.framework.model.Interface;
```

Comment:
Nevermind...I thought I saw a fully qualified "{@link com...Interface}" in the javadoc...maybe my head is still in OOTO mode... ;)